Cargo.toml

```diff
@@ -15,6 +15,11 @@ termcolor = "1.0.5"
 ansi_term = "0.12.1"
 chrono = "0.4.10"
 atty = "0.2.14"
+tracing-log = { version = "0.1", optional = true }
+log = { version = "0.4.11", optional = true }
```

Comment:
I think that `log` can be made a dev-dependency?

Reply:
I tried that, but cargo told me that dev dependencies cannot be optional...

and I just realized it doesn't need to be optional as a dev dependency :facepalm: